frontends/ol-components/src/components/Dialog/Dialog.tsx

```diff
@@ -23,7 +23,9 @@ const Header = styled.div`
 `
 
 const Content = styled.div`
-  margin: 28px 28px 40px;
+  margin: 28px;
```

Comment:
Previously, in Figma, a couple dialogs had 40px between the content and the dialog footer but most had 28px.

In THE CODE it was determined via `Dialog` vs `FormDialog`. Both used 40px margin on the content, but FormDialog also had `-12px` on the footer for an effective `28px`. This is (a) weird, and (b) **the negative margin approach did not work well with scrolling overflow.** (Which is why it's showing up in this PR).


Confirmed with @steven-hatch we want `28px` uniformly across the dialogs.

Here's an example of a noticeable change:

<img width="1362" alt="Screenshot 2024-10-15 at 11 17 04 AM" src="https://github.com/user-attachments/assets/b7e82467-adb6-4d3a-8b24-5182c86a6270">